x/group/server/msg_server.go

```diff
@@ -265,7 +266,8 @@ func (s serverImpl) CreateGroupAccount(ctx types.Context, req *group.MsgCreateGr
 			return nil, err
 		}
 
-		accountID := s.key.Derive(buf.Bytes())
+		accountDerivationKey = buf.Bytes()
```

Comment:
`buf` here is defined as the groupAccountSeq, why did we decide that? Can we use for example `accountAddr`? (this would allow us not having `bytes derivation_key = 7;` in the proto def)

Reply:
A given `groupAccountSeq` maps to a given unique group account so we're using it to generate the group account address by deriving the group root module key (`s.key`). We need to store the derivation key because then we need it to call invoke on proposal msg execution. Is this clearer?